sdcm/cluster.py

```diff
@@ -4729,7 +4729,7 @@ def install_scylla_bench(node):
             echo 'export GOPATH=$HOME/go' >> $HOME/.bash_profile
             echo 'export PATH=$PATH:/usr/local/go/bin' >> $HOME/.bash_profile
             source $HOME/.bash_profile
-            GO111MODULE=on go get -v github.com/scylladb/scylla-bench@v0.1.2
+            GO111MODULE=on go get -v github.com/scylladb/scylla-bench@v0.1.3
```

Comment:
Maybe we need to put s-b github repository in `defaults/test_default.yaml`? Not continue hold it in the code

Reply:
added